bundles/org.openhab.binding.miio/src/main/resources/database/ijai.vacuum.v19-miot.json

```diff
@@ -0,0 +1,2543 @@
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"build-map",
+				"siid":10,
+				"piid":14,
+				"friendlyName":"Map - Build Map",
+				"channel":"build_map",
+				"type":"Number",
+				"stateDescription":{
+					"readOnly":true,
+					"options":[
+						{
+							"value":"0",
+							"label":"None"
+						},
+						{
+							"value":"1",
+							"label":"Build"
+						},
+						{
+							"value":"2",
+							"label":"Clean"
+						}
+					]
+				},
+				"refresh":false,
+				"actions":[
+					
+				],
+				"readmeComment":"Value mapping [\"0\"\u003d\"None\",\"1\"\u003d\"Build\",\"2\"\u003d\"Clean\"]"
+			},
+			{
+				"property":"has-new-map",
+				"siid":10,
+				"piid":19,
+				"friendlyName":"Map - Has New Map",
+				"channel":"has_new_map",
+				"type":"Contact",
+				"stateDescription":{
+					"readOnly":true
+				},
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-enable",
+				"siid":12,
+				"piid":1,
+				"friendlyName":"Disturb - Dnd Enable",
+				"channel":"dnd_enable",
+				"type":"Contact",
+				"stateDescription":{
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-start-hour",
+				"siid":12,
+				"piid":2,
+				"friendlyName":"Disturb - Dnd Start Hour",
+				"channel":"dnd_start_hour",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":23,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-start-minute",
+				"siid":12,
+				"piid":3,
+				"friendlyName":"Disturb - Dnd Start Minute",
+				"channel":"dnd_start_minute",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":59,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-end-hour",
+				"siid":12,
+				"piid":4,
+				"friendlyName":"Disturb - Dnd End Hour",
+				"channel":"dnd_end_hour",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":23,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-end-minute",
+				"siid":12,
+				"piid":5,
+				"friendlyName":"Disturb - Dnd End Minute",
+				"channel":"dnd_end_minute",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":59,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"multi-prop-dnd",
+				"siid":12,
+				"piid":7,
+				"friendlyName":"Disturb - Multi Prop Dnd",
+				"channel":"multi_prop_dnd",
+				"type":"String",
+				"stateDescription":{
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			}
+		],
+		"experimental":true

```

Comment:
As you have tested it and it works you can remove the experimental flag (or put it to false)

Reply:
I'm not gonna lie, I haven't tested everything. I created an item based on all channels, and most of them have values. But I've no idea what 'order' is, I've not tried setting anything in map & point zones.
I mainly tested the actions (none of the "advanced actions") and the various status. Should I still set the experimental flag to false?